src/Compilers/CSharp/Portable/Symbols/Source/SourceEventDefinitionAccessorSymbol.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.CodeAnalysis.CSharp.Symbols;
+
+/// <summary>
+/// Accessor of a <see cref="SourceFieldLikeEventSymbol"/> which is a partial definition.
+/// </summary>
+internal sealed class SourceEventDefinitionAccessorSymbol : SourceEventAccessorSymbol
+{
+    internal SourceEventDefinitionAccessorSymbol(
+        SourceFieldLikeEventSymbol ev,
+        bool isAdder,
+        BindingDiagnosticBag diagnostics)
+        : base(
+            @event: ev,
+            syntaxReference: ev.SyntaxReference,
+            location: ev.Location,
+            explicitlyImplementedEventOpt: null,
+            aliasQualifierOpt: null,
+            isAdder: isAdder,
+            isIterator: false,
+            isNullableAnalysisEnabled: ev.DeclaringCompilation.IsNullableAnalysisEnabledIn(ev.CSharpSyntaxNode),
+            isExpressionBodied: false)
+    {
+        CheckFeatureAvailabilityAndRuntimeSupport(ev.CSharpSyntaxNode, ev.Location, hasBody: false, diagnostics: diagnostics);
```

Comment:
>CheckFeatureAvailabilityAndRuntimeSupport [](http://example.com/codeflow?start=8&length=41)

Are we testing cases where this check reports diagnostics?

Reply:
Adding test `InInterface`.